terror/terror.go

```diff
@@ -52,6 +57,12 @@ const (
 	CodeNoDataForHandle
 )
 
+// Variable error codes.
+const (
+	CodeUnknownStatusVar ErrCode = iota + 1
```

Comment:
CodeCommitNotInTransaction ErrCode = iota + 1
CodeUnknownStatusVar ErrCode = iota + 1

ErrorCode should be unique? @coocood